### PR TITLE
Use new app settings components

### PR DESCRIPTION
**Pull request by ma12-co**
- [x] Requires https://github.com/nextcloud/nextcloud-vue/pull/1286

### CODE REVIEW

File: src/components/SettingsDialog/SettingsDialog.vue [hunk 1]
```diff
@@ -14,103 +14,94 @@
...
+				{{ locationHint }}
+			</h3>
+			<input
+				type="text"
+				class="app-settings-section__input"
+				:value="attachmentFolder"
+				:disabled="attachmentFolderLoading"
+				@click="selectAttachmentFolder">
+		</AppSettingsSection>
+		<AppSettingsSection :title="t('spreed', 'keyboard shortcuts')">
```
Comment (nickvergessen) [inline src/components/SettingsDialog/SettingsDialog.vue:41]:
```suggestion
		<AppSettingsSection :title="t('spreed', 'Keyboard shortcuts')">
```

**Review by danxuliu — CHANGES_REQUESTED** [review]
This error appears now in the browser console when opening the main Talk UI (even if the settings dialog is not opened):
```
[Vue warn]: Error in updated hook: "TypeError: this.$el.querySelector is not a function"

found in

---> <AppSettingsDialog>
       <SettingsDialog> at src/components/SettingsDialog/SettingsDialog.vue
         <Content>
           <App> at src/App.vue
             <Root>
```

Additionally, clicking on the headers in the left area causes the right area to scroll to the appropriate section, but scrolling in the right area does not change the highlighted header in the left area (although this may be an issue of the component from nextcloud-vue, I do not know).

I have also noticed that [content in the right area can not be selected using the mouse](https://github.com/nextcloud/nextcloud-vue/issues/1505) (not that I want to, but I found it strange :-P ) and, [in Firefox, it is not possible to scroll in the right area](https://github.com/nextcloud/nextcloud-vue/issues/1506) using the arrow keys (although it works in Chromium). ~~However it also happened before using the new AppSettings components, so it could be a problem with the Modal component (just a guess).~~ Opened issues in `nextcloud-vue`.

**Comment by PVince81** [comment]
Missing padding on top:
image

**Comment by PVince81** [comment]
seems this was closed by mistake

**Comment by PVince81** [comment]
please note that nextcloud-vue was updated, so many stuff that wasn't working before is working now.
you might need to rebase

**Comment by ma12-co** [comment]
Ready for review!

**Comment by PVince81** [comment]
a wild conflict appears!

**Comment by ma12-co** [comment]
Rebased!!

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**